triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceConfig.scala

```diff
@@ -27,6 +28,11 @@ object ServiceConfig {
       .action((f, c) => c.copy(darPath = Some(Paths.get(f))))
       .text("Path to the dar file containing the trigger")
 
+    opt[Int]("http-port")
+      .required()
```

Comment:
Do we really want to require this? I think having some well-known port ala 6865 for sandbox or 7500 for the json API would be useful as a default.

Reply:
Fair. I'll go with 8080.